.github/workflows/charts.yml

```diff
@@ -0,0 +1,83 @@
+name: Charts Preprocessor
```

Comment:
Same name concern as mentioned in the schemas PR...whatever makes sense there can be updated here.